src/util/properties.rs

```diff
@@ -167,6 +172,101 @@ impl TablePropertiesCollectorFactory for MvccPropertiesCollectorFactory {
     }
 }
 
+#[derive(Clone, Debug, Default)]
+pub struct IndexHandle {
```

Comment:
For example, we may have a `RowsProperties` like:
```
struct RowsProperties {
    total_rows: u64,
    index_handles: BTreeMap<Vec<u8>, IndexHandle>,
}
```
It needs to encode the `index_handles` too, so I put the `encode_handles()` in a common place.
Besides, I prefer to put those encode/decode in one place.